addons/binding/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/communication/AreaArmRequest.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.digiplex.communication;
+
+/**
+ * Command for arming area
+ *
+ * @author Robert Michalak
```

Comment:
Please check your files for the correct author tag usage pattern that is: "name - Initial contribution"